hooks/useLocalStorage.tsx

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from 'react';
+
+export function useLocalStorage(
+  key: string,
+  defaultValue: string,
+): readonly [string | null, (val: string) => void] {
```

Comment:
Typ zwracany zbędny

Reply:
Wydaje mi się, że nie 🤔 

![image](https://user-images.githubusercontent.com/24841038/102692553-62d77a80-4214-11eb-9531-4e16761477c2.png)

Reply:
Racja, trzeba dodać to: https://github.com/typeofweb/polskifrontend/pull/25/files#r546248498